src/middlewared/middlewared/alert/source/certificates.py

```diff
@@ -82,7 +82,7 @@ async def _get_service_certs(self):
                 'type': _type,
             },
             {
-                'id': (await self.middleware.call('system.general.config'))['ui_certificate']['id'],
+                'id': (await self.middleware.call('datastore.config', 'system.settings'))['stg_guicertificate']['id'],
```

Comment:
We had an agreement to not use datastore API directly, since it might have change behind its back and its a private API.

Reply:
True that, i'll make the required changes. Thank you for pointing it out